.circleci/config.yml

```diff
@@ -129,12 +117,25 @@ jobs:
             source /tmp/venv/openfisca_core/bin/activate
             .circleci/publish-python-package.sh
 
+  check_version_and_changelog:
+    docker:
+      - image: python:3.6
```

Comment:
Here I think we should use a minimal distro containing just `git`, as https://hub.docker.com/r/alpine/git/